dts/bindings/sensor/zephyr,sensing-sensor.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2023, Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: Sensor subsystem sensor common properties bindings.
+
+include: sensor-device.yaml
+
+properties:
+  sensor-type:
+    type: int
+    required: true
+    description: sensor type id (follow HID spec definition)
+
+  sensor-index:
+    type: int
+    required: true
+    description: sensor instance index for same sensor type
```

Comment:
Previous comment was removed during rename. This should use the `chosen` DT property. Hard coding this makes for an overlay nightmare.

Reply:
According https://github.com/zephyrproject-rtos/zephyr/pull/55389/files#r1197588830, we can remove the `sensor-index` property.

Reply:
fixed, thanks.